app/auth/views.py

```diff
@@ -95,3 +116,55 @@ def login():
 		#an error occured in the server
 		response = {'message': str(e)}
 		return jsonify(response), 500
+
+@auth.route('/gettoken', methods=['post'])
```

Comment:
Nice Implementation of the send mail confirmation
Since this is an API wouldn't it be better to return a response instead of rendering a html page

Reply:
yes, it just returns a response. the template, in this case, is sent with the email.